### DISCUSSION TITLE
v2.1.2

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#2662] [Releases]
## Installation
Visit [https://getutm.app/install/][1] for the most up to date installation instructions.

## Changes
* Fixed QEMU issue causing crash running x86 Windows on M1 (thanks @kwhr0) (#2630)
* Fixed crash due to bad drive id (#2614)
* Properly delete imported/created drives on a new VM when canceled out (#2576)
* Fixed bad CPU core configuration on SPARC (#2651)
* Improvement to zh-Hant translation (thanks @pan93412) (#2595)
* Fixed an unusable CD drive always showing up in Windows x86 (thanks @conath) (#2538)
* (macOS) Fixed an issue with auto resolution changing with SPICE installed (#2587)
* (macOS) Fixed a crash when starting a VM after shutting down (#2622)
* (macOS) New global setting to (try to) only use performance cores: when set, if a VM does not manually specify a core count, instead of using the maximum number of cores on the system, UTM will use only the number of performance cores. Note this does not guarantee only the performance cores are used but only that the number of cores seen by the guest is lower. (#2557)
* (macOS) Fixed screen tearing/glitches on Windows and improved framebuffer rendering performance (#2569)
* (macOS) When SPICE tools are not properly set up, attempting to resize the window will now "revert" the size if the resolution does not get changed by the guest.
* (iOS) Fixed crash when launching with [Jitterbug][3] or other imobiledevice tools
* (iOS) Fixed Cydia install on iOS 11 (#2618)

Please check the full list on [Github][2] and help report any bug you find that is not listed.

[1]: https://getutm.app/install/
[2]: https://github.com/utmapp/UTM/issues
[3]: https://github.com/osy/Jitterbug

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v2.1.2'>v2.1.2</a>.</em>

**Comment by granddad-ru** [comment]
Installed on UTM (v.2.1.2) Windows 11 for arm64 (v.22000.65)
The video shows the whole process, starting with setting up a virtual machine:
https://youtu.be/woLAwSzE_Pk

Conclusions:
1. Windows 11 is installed in UTM from the distribution image without any problems.
2. Installing spice-guest-tools-0.164 PORTS EVERYTHING! The screen resolution immediately increases, but everything disappears on the screen and even mouse clicks do not work.
3. Upgrading UTM from version 2.1.1 to version 2.1.2 kills old virtual machines made in UTM 2.1.1. I had a working virtual machine with Windows 11 in 2.1.1, after updating to 2.1.2 it broke.

spice-guest-tools like now are not needed at all. All you need is a working virtual network card driver. The rest - sound and video work and without tools is acceptable.

Where can I get a working UTM virtual network card driver separately and how to install it?

**Comment by rdrolis** [comment]
Nice bug fix release folks. It seems the major issues I have been dealing with have been resolved. Video performance has been vastly improved on my Mac M1.
